### PR TITLE
konversation: Add version 22.08.0-1220

### PR DESCRIPTION
**Pull request by SonaliBendre**
Closes #XXXX

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/konversation.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "bin": "bin\\konversation.exe",
+    "shortcuts": [
+        [
+            "bin\\konversation.exe",
+            "Konversation"
+        ]
+    ],
+    "checkver": {
+        "url": "https://binary-factory.kde.org/job/Konversation_Release_win64/lastSuccessfulBuild/api/json",
+        "regex": "konversation-([\\d.]+-(?<build>\\d+))-windows-msvc2019_64-cl.7z"
```
Comment (issaclin32) [inline bucket/konversation.json:21]:
```suggestion
        "regex": "konversation-([\\d.]+-(?<build>\\d+))-windows-msvc(?<msvc>\\d+)_64-cl.7z"
```

File: bucket/konversation.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://binary-factory.kde.org/job/Konversation_Release_win64/$matchBuild/artifact/konversation-$version-windows-msvc2019_64-cl.7z"
```
Comment (issaclin32) [inline bucket/konversation.json:26]:
```suggestion
                "url": "https://binary-factory.kde.org/job/Konversation_Release_win64/$matchBuild/artifact/konversation-$version-windows-msvc$matchMsvc_64-cl.7z"
```

**Comment by rashil2000** [comment]
@issaclin32 the package updates daily, even when there's no commit upstream. Any way to update it only on stable releases?

**Comment by issaclin32** [comment]
> @issaclin32 the package updates daily, even when there's no commit upstream. Any way to update it only on stable releases? If not, we'll have to move this to _Versions_.

Let's move this to *Versions*. I have no way to tell which versions are "more stable".

**Pull request merged**